src/main/java/app/LocationUseCaseFactory.java

```diff
@@ -37,10 +38,12 @@ public static LocationView create(
             SuggestedLocationsViewModel suggestedLocationsViewModel,
             LocationDataAccessInterface locationDataAccessObject) {
 
+        final CardLayout cardLayout = new CardLayout();
+        final JPanel parentPanel = new JPanel();
         final LocationController locationController = createLocationUseCase(viewManagerModel, locationViewModel,
                 suggestedLocationsViewModel, locationDataAccessObject);
 
-        return new LocationView(locationViewModel, locationController);
+        return new LocationView(locationViewModel, locationController, cardLayout, parentPanel);
```

Comment:
why are we using parentPanel?

Reply:
the view requires a jpanel parameter, unsure on exact implementation. where are the cardlayout and jpanel supposed to e created.

Reply:
the view requires it because you implemented it. I am wondering what it is used for as I didn’t need to use it before.